src/components/Breadcrumb/Breadcrumb.vue

```diff
@@ -0,0 +1,157 @@
+<!--
+ - @copyright Copyright (c) 2020 Raimund Schlüßler <raimund.schluessler@mailbox.org>
+ -
+ - @author Raimund Schlüßler <raimund.schluessler@mailbox.org>
+ -
+ - @license GNU AGPL version 3 or any later version
+ -
+ - This program is free software: you can redistribute it and/or modify
+ - it under the terms of the GNU Affero General Public License as
+ - published by the Free Software Foundation, either version 3 of the
+ - License, or (at your option) any later version.
+ -
+ - This program is distributed in the hope that it will be useful,
+ - but WITHOUT ANY WARRANTY; without even the implied warranty of
+ - MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ - GNU Affero General Public License for more details.
+ -
+ - You should have received a copy of the GNU Affero General Public License
+ - along with this program. If not, see <http://www.gnu.org/licenses/>.
+ -
+ -->
+
+<docs>
+
+### General description
+
+This component is meant to be used inside a Breadcrumbs component.
+
+</docs>
+
+<template>
+	<div ref="crumb"
+		class="crumb"
+		:class="{over: hovering}"
+		draggable="false"
+		@dragstart.prevent="() => {}"
+		@drop.prevent="dropped"
+		@dragover.prevent="() => {}"
+		@dragenter="dragEnter"
+		@dragleave="dragLeave">
+		<element :is="tag" :to="to" :href="href">
+			<span v-if="icon" :class="icon" class="icon" />
+			<span v-else>{{ title }}</span>
+		</element>
+		<slot />
+	</div>
+</template>
+
+<script>
+export default {
+	name: 'Breadcrumb',
+	props: {
+		/**
+		 * The displayed title of the breadcrumb.
+		 */
+		title: {
+			type: String,
+			required: true,
+		},
+		/**
+		 * The router-link to prop [https://router.vuejs.org/api/#to](https://router.vuejs.org/api/#to)
+		 * If set, the breadcrumbs will be rendered by router-link.
+		 */
+		to: {
+			type: String,
+			default: undefined,
+		},
+		/**
+		 * Set this prop if your app doesn't use vue-router, breadcrumbs will show as normal links.
+		 */
+		href: {
+			type: String,
+			default: undefined,
+		},
+		/**
+		 * Set a css icon-class to show an icon instead of the title text.
+		 */
+		icon: {
+			type: String,
+			default: '',
+		},
+		/**
+		 * Disable dropping on this breadcrumb.
+		 */
+		disableDrop: {
+			type: Boolean,
+			default: false,
+		},
+	},
+	data() {
+		return {
+			hovering: false,
+		}
+	},
+	computed: {
+		/**
+		 * determines which element tag to use
+		 * @returns {String} the tag
+		 */
+		tag() {
+			return this.to ? 'router-link' : 'a'
+		},
+	},
+	methods: {
+		dropped(e) {
+			/**
+			 * Don't do anything if dropping is disabled.
+			 */
+			if (this.disableDrop) {
+				return false
+			}
+			this.$emit('dropped', this.to || this.href)
```

Comment:
I would probably just re-emit the original event ?
People might want to get the data differently than to/href?

Reply:
I think it is unnecessarily complicated to extract the path from the original event (if at all possible, I didn't see a nice way by quickly looking at the event). How about we emit both the event and the path like `this.$emit('dropped', e, this.to || this.href)` (and in the `Breadcrumbs` as well).

Reply:
You cannot do that, emit only take one parameter :)
Why not add to/href to the event object then?

Reply:
Interesting, when I tried it, it worked without problems. But let's just add it to the event then. Shall we do `e.path = this.to || this.href`?

Reply:
Ah sorry, yes you can! I confused myself with vuex methods/actions :)

Reply:
So let's leave it like that! https://vuejs.org/v2/api/#vm-emit